common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -230,9 +230,11 @@ def has_support(self, view, functionality):
         Returns whether the given view has support for the given functionality.
         """
         if functionality == "multi_device":
-            return self.lcp.has_multi_device_support
-        else:
-            return False
+            return all(
+                responsetypes.registry.get_class_for_tag(tag).multi_device_support
+                for tag in self.problem_types
+            )
+        return False
```

Comment:
Is the passed-in `view` parameter there because this method overrides a method with that signature? Because the `view` parameter isn't used otherwise - and the docstring seems misleading.